megamek/src/megamek/common/verifier/TestBattleArmor.java

```diff
@@ -478,9 +478,13 @@ public double getWeightStructure() {
         return getWeightChassis() + getWeightGroundMP() + getWeightSecondaryMotiveSystem();
     }
 
+    public double getWeightArmor(int trooper) {
+        return ArmorType.forEntity(ba).getWeightPerPoint() * ba.getOArmor(trooper);
+    }
+
     @Override
     public double getWeightArmor() {
-        return ArmorType.forEntity(ba).getWeightPerPoint();
+        return getWeightArmor(1);
```

Comment:
Did you mean to use a magic number here, or do we want to use "firstArmorIndex"?

Reply:
Looks like that is what I intended -_-;